programs/develop/webpack/plugin-extension/feature-web-resources/index.ts

```diff
@@ -64,7 +65,7 @@ export class WebResourcesPlugin {
               resources: resources.filter(
                 (resource) => !resource.endsWith('.map')
               ),
-              matches
+              matches: cleanMatches(matches),
```

Comment:
could you write a comment on top of this with the reasoning of why this is needed? this is for future reference.

if you're feeling adventurous, would love to have the first test for the feature-web-resources plugin. It's mainly creating a `__spec__/clean-matches.spec.ts` file with some edge cases. I'm using tests/comments as a reference to populate the new docs.

Reply:
Added a comment and some tests 👍